### ISSUE TITLE
Build docs

### ISSUE THREAD

**Issue by mattwthompson** [mdtraj/mdtraj#1810]
I don't think docs have been built for 1.9.9-1.9.10 - https://www.mdtraj.org/ is only showing something based on 1.9.8. 

Who has the keys here? Presumably somebody at ASAP has control of that URL? @sukritsingh/@jchodera?

The docs infrastructure I'm most familiar with is just throwing things on RTD and letting it automatically build new versions, optionally redirecting to/from custom domains.

More importantly, it's hard to say how well docs will build in their current state - things look [a little outdated](https://github.com/mdtraj/mdtraj/blob/65983c52473f1cbf86367ad435127a7f394e1b75/docs/requirements.txt), though I figure things can't be too far away from looking nice. I won't lie and say I've looked closely myself.

**Comment by sukritsingh** [comment]
> The docs infrastructure I'm most familiar with is just throwing things on RTD and letting it automatically build new versions, optionally redirecting to/from custom domains.

I had actually not looked closely and assumed things were already being built on RTD - I stand corrected!

I never received any keys for mdtraj.org now that you mention it...since it's FAH I would assume @xuhuihuang or @gbowman have it (Could either one of you confirm/deny that?) – I'll send an email to Xuhui and Greg if I don't hear back here

If not, @rmcgibbo do you still have the keys? Either way it'll be important to get access to it and I wasn't given much initial info.

**Comment by sukritsingh** [comment]
Update: Emailed the three of them and heard back with some details. TLDR: The account is hosted/owned separately from RTD and the CI - I'm going to look into how to transfer the accounts now into more long-term ownership/maintenance. 

 Will update as more happens!

**Comment by mattwthompson** [comment]
It'd be nice but I don't see it as a release blocker.

Good scenario: getting docs built is easy and we can make a 1.10.0 that includes whatever code changes

Bad scenario: it's more complicated and/or we keep putting it off, and also cause it to delay the 1.10.0 release

**Comment by sukritsingh** [comment]
Agreed! I'm going to see about next steps about at least getting access to the `mdtraj.org` domain as well, so that we can decide what to do with both (which will depend on how building docs goes...)

**Comment by sukritsingh** [comment]
Ok so update from that chat - and plan moving forward: 
1. Try and get RTD running is the top question. If we can get an `mdtraj.readthedocs.io` (or whatever) up and going that'll be key. Assuming RTD works, then we'll work on domain migration/redirecting appropriately. 
2. We can just set `mdtraj.org` to redirect to any nameserver, hopefully by directly pointing to RTD. If not, we can use something like Route 53 to redirect.
3. If for some reason, RTD fails then we can return to the current hosting strategy: hosting static docs on S3 and route folks there using Route 53 and CDN. Since that's how it's currently being done, this would be our fallback option. 

All of this is going forward on the assumption that our old legacy docs are unnecessary as we move forward, so we wouldn't be serving legacy docs anymore if this plan works (which I am fine with!)

**Comment by sukritsingh** [comment]
Ok so with #1888 we have working docs that can be built, which I believe gets us one step closer to resolving this. 

Once we have RTD building a "passing" stable build, we can redirect `mdtraj.org` to point to our RTD link, I think? No reason to keep the legacy docs prior from `1.9.8` or below?

**Comment by mattwthompson** [comment]
+1 on NOT keeping docs of old versions! The source code is always there if people really need to figure out how an old version works, though hopefully that's only for reproducing old results

**Comment by sukritsingh** [comment]
Fair point! And the only thing to test/benchmark against in the old stuff would be if we did a change to RMSD or something else...

In the meantime, once we have a working `mdtraj.readthedocs.io` page, I can email asking about the redirect from `mdtraj.org`

**Comment by mattwthompson** [comment]
I don't think there's been movement on the mdtraj.org domain, so I updated the repository description and README to point to the RTD page

![image](https://github.com/user-attachments/assets/8a2aa01b-dfa8-4a5e-b9d1-c0657da16edd)
